schema.sql

```diff
@@ -11,4 +11,30 @@ CREATE TABLE animals(
     weight_kg decimal
 );
 
-ALTER TABLE animals ADD COLUMN species TEXT;
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN species TEXT;
+
+CREATE TABLE owners (
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(255),
+  age INTEGER
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+ALTER TABLE animals
+	ADD PRIMARY KEY (id);
+
```

Comment:
- [ ] In this query you need to remove the column species. Please make sure to fix this query.
  - You can consider this query `ALTER TABLE animals DROP column species;`